src/Symfony/Component/String/AbstractString.php

```diff
@@ -694,6 +699,6 @@ public function __clone()
 
     public function __toString(): string
     {
-        return $this->string;
+        return $this->toString();
```

Comment:
I would keep this as is, the indirection doesn't add anything.

Reply:
Kk (I usually do this so the logic is shared)

Reply:
but there is no logic here. That's just returning the value of a property.

Reply:
but there is no logic here. That's just returning the value of a property.